### DISCUSSION TITLE
Is there a mechanism for generating a static png file to use in an image link to the player?

### DISCUSSION THREAD

**Discussion by hartzell** [asciinema/discussions#3] [Q&A]
Is there a programmatic way to generate a png image for an asciicast, like asciinema.org generates?

I'd like to be able to do something like this (from [here](https://discourse.asciinema.org/t/embed-a-recording-on-github/78)): 

```
[![asciicast](https://asciinema.org/a/170142.png)](https://asciinema.org/a/170142)
```

and have an image that shows e.g. 20 seconds into the playback.

**Comment by ku1ik** [comment]
I guess you were looking for something like https://github.com/asciinema/agg but instead of producing gif you'd like png, right?
